src/bindings/tsd.ts

```diff
@@ -213,6 +213,9 @@ export class TSDBuilder extends ExportsWalker {
       }
       sb.push("}): Promise<typeof __AdaptedExports>;\n");
     }
+    if (this.program.options.exportMemory) {
+      sb.push("export declare const memory: WebAssembly.Memory;\n");
+    }
```

Comment:
Looks like this one will only work for `--bindings esm`, whereas `--bindings raw` needs it as part of `__AdaptedExports` which is the type of exports after `instantiate`. Could be moved up perhaps, right over `this.walk()`, then adhering to `indentLevel`.